common/thorhelper/thorread.cpp

```diff
@@ -120,13 +122,13 @@ IBufferedSerialInputStream * createBufferedInputStream(IFileIO * io, const IProp
     //MORE: Is this a good idea?
     if (length == unknownFileSize)
```

Comment:
Picky: Two lines before this if() is an assignment:

offset_t length = unknownFileSize;

That is a bit confusing and could be refactored, purely for readability.